### PR TITLE
adds live plotter python script  

### PR DESCRIPTION
**Pull request by jdcaporale**
Add a live plotter example to the kodlab_mjbots_sdk. Not perfect (the plot takes time and can't read LCM in that time so there are drops) but useful functionality

### CODE REVIEW

File: python_scripts/live_motor_plotter.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+# Queues
+timestamps = collections.deque(np.zeros(WINDOWLENGTH))
+margins = collections.deque(np.zeros(WINDOWLENGTH))
+message_duration = collections.deque(np.zeros(WINDOWLENGTH))
+positions = collections.deque(np.zeros((WINDOWLENGTH,13)))
+velocities = collections.deque(np.zeros((WINDOWLENGTH,13)))
+torques = collections.deque(np.zeros((WINDOWLENGTH,13)))
+velo_filt = collections.deque(np.zeros((WINDOWLENGTH,13)))
+
+class BlitManager:
```
Comment (ethanmusser) [inline python_scripts/live_motor_plotter.py:30]: Class template pulled from https://matplotlib.org/stable/tutorials/advanced/blitting.html

**Review by ethanmusser — APPROVED** [review]

**Review by ShaneRozenLevy — CHANGES_REQUESTED** [review]
To make this really useful you should make it an abstract class that can be inherited, that way if you want to make multiple live plotting scripts you don't have to copy code. Beyond that LGTM.

**Comment by jdcaporale** [comment]
@ShaneRozenLevy  ok if I merge in as is and add an issue? Agree it would be more useful, but in case people need some live plotting now I feel like it should be available.

**Comment by ShaneRozenLevy** [comment]
Looking more closely, the `BlitManager` is general purpose. Just put it in a different file and it lgtm

**Pull request merged**